ambariclient/models.py

```diff
@@ -907,6 +907,21 @@ def execute_action(self, action, context, parameters=None, hosts=None):
         }))
         return self.request
 
+    def start(self):
```

Comment:
This should probably be on the `ClusterService` model so you can do `cluster(name).services.start()`

Reply:
Wait, that may not be right. I haven't looked at this code in a while.

Reply:
You would need to create a `ClusterServiceCollection` with this method, similar to how `HostComponentCollection` does for `HostComponent`, and then set `collection_class = ClusterServiceCollection` on `ClusterService`.

Reply:
Ah, good call. Let me fix this up and push another attempt. Thanks for the review! :)